ide/idea/workspace.xml

```diff
@@ -173,7 +183,28 @@
+      <option name="VM_PARAMETERS" value="
+                                          -Dcassandra.config=file://$PROJECT_DIR$/test/conf/cassandra.yaml
+                                          -Dcassandra.logdir=$PROJECT_DIR$/build/test/logs
+                                          -Dcassandra.reads.thresholds.coordinator.defensive_checks_enabled=true
+                                          -Dcassandra.ring_delay_ms=1000
+                                          -Dcassandra.skip_sync=true
+                                          -Dcassandra.strict.runtime.checks=true
+                                          -Dcassandra.test.flush_local_schema_changes=false
+                                          -Dcassandra.test.messagingService.nonGracefulShutdown=true
+                                          -Dcassandra.tolerate_sstable_size=true
+                                          -Dcassandra.use_nix_recursive_delete=true
+                                          -Dinvalid-legacy-sstable-root=$PROJECT_DIR$/test/data/invalid-legacy-sstables
+                                          -Djava.library.path=$PROJECT_DIR$/lib/sigar-bin
+                                          -Dlegacy-sstable-root=$PROJECT_DIR$/test/data/legacy-sstables
+                                          -Dlogback.configurationFile=file://$PROJECT_DIR$/test/conf/logback-test.xml
+                                          -Dmigration-sstable-root=$PROJECT_DIR$/test/data/migration-sstables
+                                          -XX:ActiveProcessorCount=2
+                                          -XX:HeapDumpPath=build/test
+                                          -XX:MaxMetaspaceSize=1G
+                                          -XX:SoftRefLRUPolicyMSPerMB=0
+                                          -ea
+                                          -Didea.click.expand.for.rest.of.options" />
```

Comment:
this one is new... quick google and didn't find results, what is this?

Reply:
It was an attempt to remind somebody looking at the IDEA GUI that there were options that weren't visible due to all the whitespace. As it caused confusion in review I'll just remove it.